beacon-chain/core/blocks/block_operations.go

```diff
@@ -493,13 +495,17 @@ func verifyAttestation(beaconState *pb.BeaconState, att *pb.Attestation, verifyS
 
 	// Verify that attestation.data.justified_block_root is equal to
 	// get_block_root(state, get_epoch_start_slot(attestation.data.justified_epoch)).
-	blockRoot, err := BlockRoot(beaconState, helpers.StartSlot(att.Data.JustifiedEpoch))
+	justifiedBlock, err := beaconDB.BlockBySlot(helpers.StartSlot(att.Data.JustifiedEpoch))
```

Comment:
This is not going to handle the case where that epoch start slot is a skip block. Please use the iterative lookup